azure-pipelines.yml

```diff
@@ -708,17 +708,38 @@ jobs:
     inputs:
       targetType: 'filePath'
       filePath: scripts/release/rpm/pipeline.sh
+    env:
+      IMAGE: centos7
+  - task: PublishPipelineArtifact@0
+    displayName: 'Publish Artifact: rpm'
+    inputs:
+      TargetPath: $(Build.ArtifactStagingDirectory)
+      ArtifactName: rpm-centos7
+
+
+- job: BuildRpmPackageCentOSStream8
+  displayName: Build Rpm Package CentOS Stream 8
+  pool:
+    vmImage: 'ubuntu-20.04'
+  steps:
+  - task: Bash@3
+    displayName: 'Build Rpm Package'
+    inputs:
+      targetType: 'filePath'
+      filePath: scripts/release/rpm/pipeline.sh
+    env:
+      IMAGE: centos_stream8
   - task: PublishPipelineArtifact@0
-    displayName: 'Publish Artifact: yum'
+    displayName: 'Publish Artifact: rpm'
     inputs:
       TargetPath: $(Build.ArtifactStagingDirectory)
-      ArtifactName: yum
+      ArtifactName: rpm-centos_stream8
 
 
-- job: TestYumPackage
-  displayName: Test Yum Package
+- job: TestRpmPackage
+  displayName: Test Rpm Package
   timeoutInMinutes: 120
-  dependsOn: BuildYumPackage
+  dependsOn: BuildRpmPackageCentOS7
```

Comment:
As discussed in 

- https://github.com/microsoft/azure-pipelines-yaml/issues/420
- https://github.com/microsoft/azure-pipelines-yaml/issues/20
- https://developercommunity.visualstudio.com/t/support-for-dependson-for-individual-matrix-jobs/1007464

`dependsOn` only works for the base job (all matrixes), and it can't work on a specific matrix. Since CentOS Stream 8's rpm builder is extremely slow for some reason, `TestRpmPackage` will wait for it to finish, making the whole pipeline much slower.

Here we make 2 deferent jobs:

- `BuildRpmPackageCentOS7`
- `BuildRpmPackageCentOSStream8`

so that `TestRpmPackage` can start right after `BuildRpmPackageCentOS7` finishes like before.